tonic/functional/to_bina_rep.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import math
+
+
+def to_bina_rep_numpy(
+    events: np.ndarray,
+    to_frame_transform,
+    n_frames: int = 1,
+    n_bits: int = 8,
+):
+    """Representation that takes T*B binary event frames to produce a sequence of T frames of N-bit numbers.
+    To do so, N binary frames are interpreted as a single frame of N-bit representation. Taken from the paper
+    Barchid et al. 2022, Bina-Rep Event Frames: a Simple and Effective Representation for Event-based cameras
+    https://arxiv.org/pdf/2202.13662.pdf
+    Parameters:
+        events: numpy.ndarray of shape [num_events, num_event_channels].
+        to_frame_transform (tonic.transforms.ToFrame): the ToFrame transform that creates the T*B binary event frames.
+        n_frames (int): the number T of bina-rep frames.
+        n_bits (int): the number N of bits used in the N-bit representation.
+    Returns:
+        (numpy.ndarray) the sequence of bina-rep event frames with dimensions (TxPxHxW).
+    """
+    assert "x" and "t" and "p" in events.dtype.names
```

Comment:
Why is `y` not checked?

Reply:
I took this assert from the `to_frame_numpy()` function, that works with 1-dim input like audio record (if I understood well). I thought I could reuse this ability of `ToFrame` in my implementation, but I just figured out that my code cannot adapt to 1-D events. I will add the assert in the next commit.